content/documentation/pages/5-feature-guides/1-streams/5-stream-application-dsl.md

```diff
@@ -4,4 +4,75 @@ title: 'Stream Application DSL'
 description: 'How to use the Stream Application DSL'
 ---
 
-Introduction to the Stream Application DSL.
+# Stream Application DSL
+
+The Stream Pipeline DSL described in the previous section automatically sets the input and output binding properties of each Spring Cloud Stream application.
+This can be done because there is only one input and/or output destination in a Spring Cloud Stream application that uses the provided binding interface of a `Source`, `Processor`, or `Sink`.
+However, a Spring Cloud Stream application can define a custom binding interface such as the one shown below
+
+```java
+public interface Barista {
+
+    @Input
+    SubscribableChannel orders();
+
+    @Output
+    MessageChannel hotDrinks();
+
+    @Output
+    MessageChannel coldDrinks();
+}
+```
+
+or as is common when creating a Kafka Streams application,
+
+```java
+interface KStreamKTableBinding {
+
+    @Input
+    KStream<?, ?> inputStream();
+
+    @Input
+    KTable<?, ?> inputTable();
+}
+```
+
+In these cases with multiple input and output bindings, Data Flow cannot make any assumptions about the flow of data from one application to another.
+Therefore the developer needs to set the binding properties to 'wire up' the application.
+The _Stream Application DSL_ uses a `double pipe`, instead of the `pipe symbol`, to indicate that Data Flow should not configure the binding properties of the application. Think of `||` as meaning 'in parallel'.
+
+For example:
+
+`dataflow:> stream create --definition "orderGeneratorApp || baristaApp || hotDrinkDeliveryApp || coldDrinkDeliveryApp" --name myCafeStream`
+
+A graphical representation of the stream woud look similar to the following:
+
+![Stream Application DSL](images/stream-application-dsl.png)
+
+<!--NOTE-->
```

Comment:
Let's remove the note. The Microsite is built on top of 2.x minimum. Besides that, 1.x line is strictly on maintenance only mode.